### PR TITLE
pcre2: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by jonchang**

### CODE REVIEW

**Comment by maxim-belkin** [comment]
```
** Cannot --enable-pcre2grep-libz because zlib.h was not found
```

**Comment by maxim-belkin** [comment]
requesting a review from @sjackman because of added dependency

File: Formula/pcre2.rb [hunk 1]
```diff
@@ -14,6 +15,7 @@ class Pcre2 < Formula
   end
 
   depends_on "bzip2" unless OS.mac?
+  depends_on "zlib" unless OS.mac?
```
Comment (sjackman) [inline Formula/pcre2.rb]: Preferred style is
```ruby
unless OS.mac?
  depends_on "bzip2"
  depends_on "zlib"
end
```

File: Formula/pcre2.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# pcre2: Build a bottle for Linuxbrew
```
Comment (sjackman) [inline Formula/pcre2.rb]: Remove this commit using `git rebase -i`

**Comment by sjackman** [comment]
Apart from the minor style fixes above, looks good to me.

**Pull request closed without merging**